packages/server/src/api/routes/ai.ts

```diff
@@ -0,0 +1,9 @@
+import Router from "@koa/router"
+import * as controller from "../controllers/ai"
+import { auth } from "@budibase/backend-core"
+
+const router: Router = new Router()
+
+router.post("/api/ai/js", auth.builderOrAdmin, controller.generateJs)
```

Comment:
Should this not be builder only?

Reply:
I'm not sure to be honest. Why not admin as well?

Reply:
You are right, this is on the app scope!